provider/cluster_rosa_classic_resource.go

```diff
@@ -57,6 +57,7 @@ const (
 	maxClusterNameLength = 15
 	tagsPrefix           = "rosa_"
 	tagsOpenShiftVersion = tagsPrefix + "openshift_version"
+	lowestHttpTokensVer  = "4.11.0"
```

Comment:
What if the cluster version will be set to 4.10? (the MinVersion of cluster version is "4.10")

Reply:
I just thought we would like to be aligned with the minimum version 
But I think it's fine to handle it like that